shared/package.json

```diff
@@ -123,7 +123,7 @@
     "glamorous": "4.12.0",
     "glamorous-native": "1.3.0",
     "iced-runtime": "1.0.3",
-    "immutable": "git://github.com/keybase/immutable-sorted#58fa578bebcf19da76a9575655c4d6ec6f72762d",
+    "immutable": "git://github.com/keybase/immutable-sorted#2a5895d4a5b36e632f833aec2cc84d2ad319bf71",
```

Comment:
pulls in some small flow typing fixes that makes withMutations and updateIn with records type correctly